src/syscalls/deprecated_syscall_request.rs

```diff
@@ -228,6 +246,13 @@ impl From<DeprecatedReplaceClassRequest> for DeprecatedSyscallRequest {
 //  FromPtr implementations
 // ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+/// We are implementing DeprecatedFromPtr trait for various types
```

Comment:
This is not an implementation, this is just a trait definition. Removing this first line of the comment and leaving:
```
/// This trait provides functionality to convert from a raw pointer
/// to a specific deprecated system call request
```
should be enough.